src/style.css

```
@@ -0,0 +1,84 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  flex-basis: 100%;
+  height: 300px;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  align-items: center;
+}
+
+.block:nth-child(1) {
+  background-color: rgb(255, 0, 0);
+}
+
+.block:nth-child(2) {
+  background-color: rgb(200, 4, 4);
+}
+
+.block:nth-child(3) {
+  background-color: rgb(150, 4, 4);
+}
+
+.block:nth-child(4) {
+  background-color: rgb(100, 4, 4);
+}
+
+.block:nth-child(5) {
+  background-color: rgb(50, 4, 4);
+}
+
+.block:nth-child(6) {
+  background-color: #000;
+}
+
+/* @media (min-width: 900px) {
+  .block {
+    flex-basis: 33.3%;
+  }
+  .block:nth-child(6) {
+    order: 1;
+  }
+  .block:nth-child(5) {
+    order: 2;
+  }
+  .block:nth-child(4) {
+    order: 3;
+  }
+} */
+
+@media (min-width: 600px) {
+  .block {
+    flex-basis: 50%;
+  }
+  .block:nth-child(1),
+  .block:nth-child(2),
+  .block:nth-child(4) {
+    order: -1;
+  }
+}
+
+@media (min-width: 900px) {
+  .block {
+    flex-basis: 33.3%;
```

```suggestion
    flex-basis: 33.33%;
```
For now there is a visible white line on the right side of the screen

https://user-images.githubusercontent.com/38065883/170696048-467a2cfb-9a1f-4c1f-8662-bc959afadc25.mov